### PR TITLE
Add Jwt with AsymmetricKey scenario

### PR DESCRIPTION
**Pull request by BrennanConroy**
Closes https://github.com/aspnet/Benchmarks/issues/1880

### CODE REVIEW

File: build/mvc-scenarios.yml [hunk 1]
```diff
@@ -74,6 +74,8 @@ parameters:
     arguments: --scenario ApiCrudListProducts $(crudApiJwtJobs) --property scenario=ApiCrudListProductsJwtAuth
   - displayName: plaintext (JWT)
     arguments: --scenario NoMvcAuth $(crudApiJwtJobs) --property scenario=PlaintextRawJwt
+  - displayName: plaintext (JWT Asymmetric Key)
+    arguments: --scenario NoMvcAsymmetricAuth $(crudApiJwtJobs) --property scenario=PlaintextRawJwt
```
Comment (sebastienros) [inline build/mvc-scenarios.yml:78]:
```suggestion
    arguments: --scenario NoMvcAsymmetricAuth $(crudApiJwtJobs) --property scenario=PlaintextAsymmetricKey
```

**Review by sebastienros — APPROVED** [review]
Pending changes

File: build/mvc-scenarios.yml (hunk 1 above)
Comment (BrennanConroy) [reply build/mvc-scenarios.yml:78]: Should it be "PlaintextJwtAsymmetricKey"?

Comment (sebastienros) [reply build/mvc-scenarios.yml:78]: It's just the text that appears in the dashboard, if it's a better description then yes.

**Pull request merged**